public/src/components/channelManagement/bannerTests/bannerTestVariantsEditor.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { useState, useEffect } from 'react';
-import { Theme, createStyles, WithStyles, withStyles } from '@material-ui/core';
-import { BannerVariant } from './bannerTestsForm';
+import React, { useEffect, useState } from 'react';
+import { createStyles, Theme, WithStyles, withStyles } from '@material-ui/core';
+import { BannerTemplate, BannerVariant } from './bannerTestsForm';
```

Comment:
Do you have something to auto sort alphabetically? If so hit me up! 🔥

Reply:
must be intellij!

Reply:
I'm jealous!